File: examples/hello.py
# Python 3 — Example for Hypervisor VM
# Run: docker_run.bat examples\hello.py

def greet(name):
    return f"Hello from Python 3 on Linux! 🐍  → {name}"

numbers = [1, 2, 3, 4, 5]
total = sum(numbers)
squared = [x**2 for x in numbers]

print(greet("HypervisorVM"))
print(f"Sum of {numbers} = {total}")
print(f"Squares: {squared}")
print(f"Running on Linux inside Docker! 🐧")

File: examples/hello.cpp
// C++ — Example for Hypervisor VM
// Run: docker_run.bat examples\hello.cpp

#include <iostream>
#include <vector>
#include <algorithm>
#include <string>

using namespace std;

int main() {
    cout << "Hello from C++ on Linux! 🐧" << endl;
    cout << "Running inside Docker Hypervisor!" << endl << endl;

    vector<int> nums = {5, 2, 8, 1, 9, 3, 7, 4, 6};
    cout << "Original: ";
    for (int n : nums) cout << n << " ";
    cout << endl;

    sort(nums.begin(), nums.end());
    cout << "Sorted:   ";
    for (int n : nums) cout << n << " ";
    cout << endl;

    string msg = "HypervisorVM";
    reverse(msg.begin(), msg.end());
    cout << "\nReversed string: " << msg << endl;

    return 0;
}
